core/drivers/crypto/caam/cipher/local.h

```diff
@@ -44,6 +53,20 @@ struct cipherdata {
 	struct caambuf ctx;          /* CAAM Context Register */
 	struct caamblock blockbuf;   /* Temporary Block buffer */
 	const struct cipheralg *alg; /* Reference to the algo constants */
+
+	/* Additionnal Data for the MAC */
+	unsigned int mode; /* MAC TEE_CHAIN_MODE* */
+	size_t countdata;  /* MAC Number of input data done */
```

Comment:
"Number of input data" or "Number of input bytes"?

Reply:
```Number of input data``` is better